pkg/apis/psp/v1beta1/doc.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/psp
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/extensions
```

Comment:
@sttts I'm not sure whether I need this line or not. My theory is that it should generate conversions from/to extensions to psp types but I'm not sure. Could you tell me what it does?

Reply:
From Makefile.generated_files:
```
#
# Conversion generation
#
# Any package that wants conversion functions generated must include one or
# more comment-tags in any .go file, in column 0, of the form:
#     // +k8s:conversion-gen=<CONVERSION_TARGET_DIR>
#
# The CONVERSION_TARGET_DIR is a project-local path to another directory which
# should be considered when evaluating peer types for conversions.  Types which
# are found in the source package (where conversions are being generated)
# but do not have a peer in one of the target directories will not have
# conversions generated.
```

Reply:
I expect that you don't need this if all your types are in the new pkg/apis directory. But if you share the internal types with pkg/apis/extensions, you will need this tag.